modules/lambda/variables.tf

```diff
@@ -71,16 +71,16 @@ variable "secrets_arns" {
   default     = []
 }
 
-variable "store_secret_in_secrets_manager" {
-  description = "Whether to store your secrets in secrets manager, default is false"
-  type        = bool
-}
-
 variable "lambda_source_code_filename" {
   type        = string
   description = "The lambda source code filename"
 }
 
+variable "lambda_source_code_path" {
```

Comment:
You can use terraform-docs to generate the readme file for the lambda module.
this is relevant for all the modules